.github/workflows/test.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright 2024 Hewlett Packard Enterprise Development LP
+name: Test
+
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        go: [ '1.21' ]
```

Comment:
Is there much reason to use a matrix here?

Additionally this version is lower than what is specified in the `go.mod` file. This will likely cause the compiler to download the version of Go specified in the `go.mod` file

Reply:
updated to 1.22.3. Matric may be of limited use, but no harm either